frontend/test/metabase/scenarios/native-filters/helpers/e2e-field-filter-helpers.js

```diff
@@ -28,6 +28,7 @@ export function setWidgetType(type) {
 export function addWidgetStringFilter(value) {
   popover()
     .find("input")
+    .first()
```

Comment:
That will do for now. Thank you.

In general, I like to avoid using ambiguous pointers like `first`, `last`, etc. But without going through every single test where `addWidgetStringFilter` is used in order to find an optimal solution, this is ok.

At the first glance, if other input fields are "checkboxes" then we could've solved it by scoping the input field to its type: `.find("input[type=text]")`

Reply:
I tried but well it fails a lot of EE, so I'll keep the .first() : https://github.com/metabase/metabase/runs/7183745277?check_suite_focus=true